renku/core/models/datasets.py

```diff
@@ -540,20 +540,21 @@ def __attrs_post_init__(self):
             )
 
 
-def is_dataset_name_valid(name, safe=''):
-    """A valid name is a valid Git reference name with no /."""
-    # TODO make name an RFC 3986 compatible name and migrate old projects
+def is_dataset_short_name_valid(short_name, safe=''):
```

Comment:
Can we remove `safe`? it's not actually used.

Reply:
Done!